base/regex.jl

```diff
@@ -164,19 +164,28 @@ See [`keys`](@ref keys(::RegexMatch)) for more information.
 
 # Examples
 ```jldoctest
-julia> m = match(r"(?<hour>\\d+):(?<minute>\\d+)(am|pm)?", "11:30 in the morning")
+julia> m = match(r"(?<hour>\d+):(?<minute>\d+)(am|pm)?", "The time is 11:30")
```

Comment:
The double backslashes are needed here since this is inside a julia source file. What is the purpose of changing the string to be matched?

Reply:
In the REPL the string to be matched is too long causing it to span multiple lines:

```julia
  julia> m = match(r"(?<hour>\d+):(?<minute>\d+)(am|pm)?", "11:30 in the morning
")
  RegexMatch("11:30", hour="11", minute="30", 3=nothing)
```

I used a shorter one, since it doesn't change anything and keeps everything clean on one line.

About the double backslashes, yeah thank you. I need to add them.

Reply:
I mean, that is just because you have a narrow terminal then. As a general comment for contributions, it is preferred not to make stylistic/opinionated/drive-by changes like this, it creates noise and makes it more difficult to review (i.e. in this case I was thinking for a while why this change was made and why it was necessary for the other additions or not).

Reply:
Oh okay. I get that now 👍